Kconfig

```diff
@@ -0,0 +1,99 @@
+mainmenu "IOT2050 Image Configuration"
+
+config KAS_BUILD_SYSTEM
+	string
+	default "isar"
+
+choice
+	prompt "Image type"
+	default IMAGE_EXAMPLE
+
+config IMAGE_EXAMPLE
+	bool "Example image"
+	help
+	  This is the official example image with several tools as well as
+	  Node-RED preinstalled. It comes without a graphical user interface.
+
+config IMAGE_LXDE
+	bool "LXDE example image"
+	help
+	  Based on the example image, this adds an LXDE-based graphical user
+	  interface.
+
+config IMAGE_BOOT
+	bool "Firmware image"
+	help
+	  Build the firmware image that is responsible for booting the device.
+
+	  WARNING: Do not flash this image onto your device unless you know
+	  that it fits AND you have an external flash programmer at hand that
+	  allows to recover. Otherwise you risk to BRICK THE IOT2050!
```

Comment:
Seems this should be adapted to the pg1&pg2 split.

Reply:
right, will have a look